tests/resources/dockerfiles/docker-pull-mitm-proxy/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Copyright 2016-2017 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+FROM python:3
+COPY archive.tar /
+COPY docker-mitm.py /
+COPY mitmdump /
+RUN chmod +x /mitmdump
```

Comment:
just check it in to the repo as executable?

Reply:
Sure